packages/host/app/resources/realm.ts

```diff
@@ -0,0 +1,108 @@
+import { service } from '@ember/service';
+import { tracked } from '@glimmer/tracking';
+
+import { restartableTask } from 'ember-concurrency';
+import { Resource } from 'ember-resources';
+import window from 'ember-window-mock';
+
+import { type TokenClaims } from '@cardstack/runtime-common';
+
+import type MatrixService from '@cardstack/host/services/matrix-service';
+
+interface Args {
+  named: {
+    realmURL: URL;
+  };
+}
+
+type RealmJWT = TokenClaims & { iat: number; exp: number };
```

Comment:
We already have this type, we can import it from here:

https://github.com/cardstack/boxel/blob/da8392578752aafbedd34427ee017b6ea6dd0426/packages/runtime-common/realm-auth-client.ts#L6